www/js/types/diaryTypes.ts

```diff
@@ -96,7 +102,7 @@ export type CompositeTrip = {
   confirmed_trip: ObjectId;
   distance: number;
   duration: number;
-  end_confirmed_place: BEMData<ConfirmedPlace>;
+  end_confirmed_place: ConfirmedPlace;
```

Comment:
Curious about what this change does. Why did we have BEMData before and why don't we need it now?

Reply:
It was just incorrect. I noticed because Typescript complained when I accessed properties on the place objects while making the changes

> Updated types:
> 
> -Since the CompositeTrip type definition is based on the return type of Timeline.readAllCompositeTrips, which is after "unpacking", start_confirmed_place and end_confirmed_place are simply ConfirmedPlace, not BEMData<ConfirmedPlace>
> -ConfirmedPlace.location is a `Point`, (having `coordinates`) not just a generic `Geometry`
> -added geocoded_address and reverse_geocode, both as optional since it depends on e-mission/e-mission-server#973